content/team/mauricio-salatino/_index.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Mauricio Salatino
+roles: ["guest"]
+skills: "Knative"
+location: ""
+
+# Social Media
+twitter: "salaboy"
+website: "https://salaboy.com/"
+facebook: ""
+linkedin: ""
+github: "salaboy"
+pinterest: ""
+instagram: ""
+youtube: ""
+---
+
+Mauricio is currently working at the [Knative OSS Project](https://knative.dev/) as part of the VMware MAPBU (Modern Apps Platform Business Unit). 
```

Comment:
We don't call it MAPBU anymore, right? It's MAMBG? Also, maybe just say "as part of VMware."?

Reply:
It is done!
\(^-^)/